lhotse/bin/modes/workflows.py

```diff
@@ -55,6 +56,36 @@ def workflows():
 @click.option(
     "-d", "--device", default="cpu", help="Device on which to run the inference."
 )
+@click.option(
+    "--faster-whisper",
+    is_flag=True,
+    default=False,
+    help="If True, use faster-whisper's implementation based on CTranslate2.",
+)
+@click.option(
+    "--faster-whisper-use-vad",
+    is_flag=True,
+    default=False,
```

Comment:
Should we enable it as default / is there a reason not to?

Reply:
It's useful for longer recordings with silence/noise portions, but can actually slow down processing of relatively short recordings (such as librispeech, which is already chunked). I assume the primary use case for this workflow is processing/annotating longer files, so it would make sense to enable it as default. If so, we should disable the `--faster-whisper-add-alignments` option when VAD is active, as it interacts badly with faster-whisper's word timestamps feature (see e.g. https://github.com/guillaumekln/faster-whisper/issues/125).

Reply:
Sounds good to me! Can you make the relevant changes? 

Also maybe we can emit a warning that says what you just wrote if somebody turns on both VAD and alignment options.